src/osm_flex/simplify.py

```diff
@@ -0,0 +1,43 @@
+"""
+This file is part of OSM-flex.
+Copyright (C) 2023 OSM-flex contributors listed in AUTHORS.
+OSM-flex is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+OSM-flex is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+-----
+simplification functions
+"""
+
+def remove_small_polygons(gdf, min_area):
+    """Remove (multi-)polygons of area smaller than min_area
+    Points and lines are untouched.
```

Comment:
would be nice to have the option of a metres=True kwarg (similar to the CLIMADA interpolation funcs), where min_area can hence be specified in m2, irrespective of the original projection of the gdf.
Requires a check & potential re-projection of the gdf under the hood.

Reply:
I think that would be nice for a future extension. Shall we already merge this first version?

Reply:
hmm, I'd add this right away maybe, so we don't have to change the signature of the function later on